### DISCUSSION TITLE
100% Carrier Sense (vermutlich zufällig seit Update)

### DISCUSSION THREAD

**Discussion by theimo1221** [OpenCCU/OpenCCU#1584] [🇩🇪 Fragen&Antworten] [answered]
![image](https://user-images.githubusercontent.com/20605452/145114926-cd7fff89-76f0-418f-98db-392598bd15bc.png)

Nabend zusammen, 

bin heute erst auf die neuste Version gewechselt (ich weit etwas verspätet.... viel los zur Zeit...) und folgendes ist passiert:
1. Update ganz normal eingespielt, frisch hochgefahren alles gut
2. Firmware Update für HmIp-HAP heruntergeladen
3. Firmware Update ins HmIp-HAP eingespielt
--> Hiernach ging der Carrier Sense beider HAP's auf 100%
4. Ich mir gedacht okay, mag vielleicht sein, wenn die beiden HAP's unterschiedliche Versionen haben, dass die sich behakeln
--> Reboot System und Update in zweiten HAP eingespielt
5. Carrier Sense blieb oben auch nach mehrmaligen neustart der VM und des Servers
6. Alle Batterien mit weniger als 2,6V getauscht, zusätzlich sogar aus Vorsicht alle Heizungsaktoren
--> Keine Veränderung
7. Backup eingespielt und nochmal Update durchgeführt
--> Keine Veränderung
8. Alle Strombetriebenen Geräte einmal gekappt und neu gestartet (Sicherung) sowie alle Batteriebetriebenen einmal resetet
--> Keine Veränderung
**9. Nachtrag: Frische USB Platine (HB-RF-USB von Alex) gelötet und angeschlossen**
--> Keine Veränderung

Komisch ist: Daten meiner Sensoren/Aktoren kommen an, aber scheinbar gehen Schaltbefehle nicht durch und auch Acknowledges kommen nicht an.

Vielleicht hat bei mir die Platine kein Vernünftiges Update bekommen, vielleicht würde auch c18f06e6312ec187f5b556a9a82a3b1b943e1f6b helfen. (@jens-maus könntest du mir bitte einen nightly/snapshot mit ova builden?)

Mal schauen, wie/ob ich morgen die Rollos hochkriege....

Viele Grüße
 
Thiemo  

EDIT: Einen Ova Build kriege ich bei mir irgendwie nicht gebaut.
![image](https://user-images.githubusercontent.com/20605452/145192948-cb916fba-3f3b-4de2-9fe3-acdf66a2d2d4.png)

**Comment by Baxxy13** [comment]
Sieht ja böse aus.
Was mir aber unklar ist...
Du schreibst von... 

> USB Platine

Was ist damit gemeint? Ein HmIP-RFUSB oder die HB-RF-USB(-2) Platinen von Alex?

Wenn ersteres, wie hast du es dann geschafft 2 HAP's zu integrieren? Das sollte doch mit der alten RFUSB-Firmware gar nicht möglich sein.
Und wieso brauchst du einen extra .ova Nightly? Die Beta-Firmware für den HmIP-RFUSB ist doch schon seit 23.11. also auch im letzten Nightly (03.12.) drin.

**Reply by theimo1221** [reply]
@Baxxy13 Danke für deine Antwort.

Jo ich meine die von Alex: HB-RF-USB  (https://smartkram.de/shop/produkte/bauteile-zentralen/platine-hb-rf-usb-tk/)

In dem Nighlty (https://github.com/jens-maus/RaspberryMatic/actions/runs/1533139403) ist aber keine .ova Version, oder welche Version meinst du, bzw. wo wie kann ich die neue Firmware Version für den RFUSB direkt nutzen? Es könnte sein, dass mir dieser Checkin hier helfen würde: c18f06e

**Reply by jens-maus** [reply]
> Jo ich meine die von Alex: HB-RF-USB (https://smartkram.de/shop/produkte/bauteile-zentralen/platine-hb-rf-usb-tk/)
> In dem Nighlty (https://github.com/jens-maus/RaspberryMatic/actions/runs/1533139403) ist aber keine .ova Version, oder welche Version meinst du, bzw. wo wie kann ich die neue Firmware Version für den RFUSB direkt nutzen? 

Stop mal. Du verwechselst hier `HB-RF-USB` mit `HmIP-RFUSB`. Das sind zwei Paar Schuhe. Ergo brauchst du auch kein nightly build wo eine neuere Firmware des `HmIP-RFUSB` drin ist.

**Reply by theimo1221** [reply]
Okay...
Unterwegs ist noch eine Platine um mit AskSinAnalyzerXS zu schauen ob/falls jemand anderes funkt....
Keine Ahnung, was ich sonst noch tun könnte...

**Reply by jens-maus** [reply]
> Okay... Unterwegs ist noch eine Platine um mit AskSinAnalyzerXS zu schauen ob/falls jemand anderes funkt.... Keine Ahnung, was ich sonst noch tun könnte...

Bin mir nicht sicher ob dir das Helfen wird. Ein solche extremer Carrier Sense deutet daraufhin das in der näheren Umgebung da irgendetwas das 868MHz Band so "zuballert" das da nichts mehr geht bzw. bei solch einem CarrierSense sendet das Funkmodul eben gar nichts mehr raus wegen Listen-Before-Talk. Scheint da also irgendein Störsender in der Umgebung zu geben. Einfach mal alle deine Funksensoren/Aktoren mal ausser kraft setzen und beobachten ob irgendwann der CarrierSense runter geht. Wenn nicht hat dein Nachbar vielleicht ne kaputte Mikrowelle die auch bei offener Türe arbeitet oder er bastelt selbst irgendwas im 868MHz Band rum.

**Reply by theimo1221** [reply]
Ich habe keins der Geräte (Basis und AP's) anders positioniert und persönlich im Haus auch nichts Neues in Betrieb genommen.
Wegen dem stromlos Schalten ganzer Stockwerke hier kann ich eigentlich alle Geräte am Strom ausschließen (außer es sind mehrere Geräte die hier aktuell zugleich Amok laufen).

Gefühlt muss es irgendwie durch/nach dem Update und ggf. den damit verbundenen Firmware-Updates der HAP's zu tun haben. Aber selbst wenn ich beide Ap's rausziehe, dann ist der Carrier-Sense der Hauptzentrale trotzdem auf 100%.....
Ansonsten muss es ein extremer Zufall sein, dass just dann ein Nachbar einen kräftigen Sender aktiviert hat. Vor Allem müsste der ja verdammt stark sein, um bei mir die Zentrale im Keller + 2 AP's im 1. Stock zugleich lahm zu legen. Wir haben rechts/links vom Haus auch noch jeweils eine Garage + Garage Nachbarn und dann erst das Nachbarhaus, also ist da schon was an Abstand....

**Reply by fastbag** [reply]
Bei mir ist das gleiche passiert, nur habe ich es home assistant laufen als plugin.
Nur bei mir hat er nach dem update fast alle Rollos als nicht mehr erreichbar. Auch das zurückspielen des Backups hat keine 
Veränderung bewirkt weder beim CarrierSense 100% noch das er die Gerät wieder findet.
Vorher lief alles ohne Probleme ausser das nach dem umstieg auf Pugin(von Raspi4), der HmIP-HAP nicht mehr ging.
![image](https://github.com/jens-maus/RaspberryMatic/assets/46375317/7859d79f-1917-4fc6-a075-40b1a5a80d0e)

**Comment by Baxxy13** [comment]
Ich hole mir die Nightly's immer von hier: [Experimental Nightly Build Snapshots](https://github.com/jens-maus/RaspberryMatic/releases/tag/snapshots)

Wenn 3 Funkempfänger  (RPI-RF-MOD +2x HAP) 100%CS anzeigen muss aber heftig was stören.
Da wird dir der Analyzer vermutlich nicht wirklich weiterhelfen. 
Ich würde tatsächlich mal alle HM/IP Geräte gleichzeitig stromlos machen und gucken.
Besteht das Problem dann weiterhin dann zusätzlich die ganze Elektronik im Haus entstromen (außer Zentrale und HAP's)
Und so Schritt für Schritt vorgehen um möglichst viel auszuschließen.

**Reply by theimo1221** [reply]
![image](https://user-images.githubusercontent.com/20605452/145248167-30a7fefd-904b-4fe5-bcbc-88765f3e914f.png)

Mit dem neusten Nightly bleibt zumindest die Zentrale (Keller) auf 90%, also müsste es im EG oder höher sein...
Ich werde euch auf dem Laufenden halten

**Reply by jens-maus** [reply]
Bei solch groben Funkproblemen hilft dir auch kein Firmware Update/downgrade. Du hast hier definitiv einen massiven Funkstörer in der nächere Umgebung. Da hilft wirklich nur alle deine HmIP Geräte stromlos zu machen (auch die Batteriebetriebenen) und dann abzuwarten. Wenn der CS dann immer noch so hoch bleibt muss es ein drittgerät bzw. der Nachbar sein. Und wenn es der Nachbar ist hilft da ohnehin nur einen Frequenzanalyzer besorgen und mit einem DVB-T Stick dir das Frequenzband anzuschauen oder gleich die Störungsstelle der Bundesnetzagentur anzurufen und die kommen mit dem Sendewagen und versuchen die Störung ausfindig zu machen. Kann aber natürlich auch sein das du irgendwo selbst einen weiteren Störer (RaspberryPi4 oder so hast der wie wild das Frequenzband dicht macht).

**Comment by theimo1221** [comment]
Nabend @jens-maus und @Baxxy13,

wie von euch empfohlen habe ich alle meine HmIp stromlos geschaltet (Batterie raus) sowie das gesamte Haus vom Strom getrennt.
Zusätzlich habe ich auch akkubetriebene Elemente wie DECT Telefone, Staubsaugerroboter, batteriebetriebene Kameras alles ausgeschaltet.
Auch den zweiten Server mit ioBroker und Zigbeemodul habe ich abgeschaltet.

Es liefen also einzig: 1xUSV, 1xRouter, 1xHauptserver mit RaspMatic.
Theoretisch waren noch kleinere batteriebetriebene Zigbee Sensoren "an", aber an sich ist das ja ein ganz anderes Frequenzband und ich weiß nicht ob Geräte wie ein kleiner Aqara Water Sensor außem Keller auf einem völlig anderen Frequenzband auch den 1. Stock lahm legen....

Der Carrier Sense lag leider dennoch mehrere Minuten (ca. 10) auf 100% auch nach einem Neustart der Zentrale ging der nahezu sofort über 90% wieder auf 100% hoch.

Anschließend habe ich meine USV im Büro eingeschaltet und der damit dort hochgefahrene AP war anschließend auf 100% Carrier Sense.
 

Ich bin mal gespannt auf das Messgerät (hoffe es kommt bald) und werde dann damit auch mal an der Straße langlaufen, ob es in die eine oder andere Richtung der Nachbarn kräftiger wird..... 
Im Notfall werde ich alle meine HmIp Aktoren/Sensoren/Taster durch weitere Zigbee Geräte ersetzen müssen (meine Frau will die Rollos wieder hochfahren können tagsüber 😄) , dann bräuchte ich für meine HmIp-Sachen aber einen Großabnehmer....

Viele Grüße

Thiemo

**Reply by Baxxy13** [reply]
Ich drücke die Daumen das du schnell rausbekommst was das verursacht und es dann auch zeitnah beseitigen kannst.

**Reply by theimo1221** [reply]
![image](https://user-images.githubusercontent.com/20605452/145358084-3a1af08e-82de-46b4-a0ef-be3e7e41ec5c.png)

![image](https://user-images.githubusercontent.com/20605452/145358302-d64e04f3-3e4b-4ee5-81cf-f79d09c1b3b5.png)

Moin zusammen,

heute morgen scheint der Spuk fast komplett weg zu sein??!
Ich habe aber seit dem Abschalten gestern Abend und einem Prüfen des CS gegen 0 Uhr nichts mehr gemacht/geändert bis heute morgen......

**Reply by jens-maus** [reply]
Tja, dann hat der Nachbar seine Funkkanone (Babyphone, etc.) eben stromlos gemacht oder das Gerät von ihm das so stark die Funkbänder dich gemacht hat (weil es ggf. eben gerade am sterben war) hat nun endlich den Geist komplett aufgegeben. :)

Halten wir aber fest (weil das Topic das suggeriert): Es liegt NICHT an 3.61.5 und auch an sonst keinem Update, Firmware, etc. sondern ist eine lokale Sache die nur in deiner Installation bzw. Umgebung zu suchen ist. Hochfrequenztechnik ist eben nunmal Störungsanfällig und da hilft auch nicht dann vermeintlich komplett auf Zigbee umsteigen zu wollen, denn da kann dir genau das selbe passieren wenn der Nachbar ein kaputtes Gerät hat das wie wild vor seinem nahendem Ende rumfunkt oder die letzte Sonneneruption ihren Strahlenfokus zufälligerweise genau auf dein Grundstück niederlässt. Unter'm Strich hilft da nur dauerhaft auf Kabel (HmIP-Wired & Co) umzusteigen wenn du solche HF-Störung gänzlich ausschließen willst. Bei Zigbee hättest du das z.B. nicht mal mitbekommen weil es da keine CarrierSense, geschweigedenn eine DutyCycle Anzeige gibt.

**Reply by theimo1221** [reply]
> Bei Zigbee hättest du das z.B. nicht mal mitbekommen weil es da keine CarrierSense, geschweigedenn eine DutyCycle Anzeige gibt.

Wenn nicht mitbekommen bedeutet hätte alle Geräte haben zwar zu kämpfen aber senden/erhalten Kommandos wäre das ja okay wenn ich davon nichts merke 😄 
Bei mir kamen ja noch Signale der einzelnen Geräte an der Zentrale an, die hat allerdings meinem Verständnis nach wegen hohem CS nie Antworten/Kommandos rausgesendet. Ginge da theoretisch eine Einstellung die sagt "Ignoriere Carrier Sense und versuch es ggf. trotzdem?"
Das Verhalten ist ja bei z.B. Zigbee ja ähnlich, da ignorieren m.W.n die einzelnen Kommunikationsteilnehmer ja auch einfach das Rauschen an Kommunikation was sie nicht betrifft und bei harten Kollisionen senden sie gemäß IEEE 802.15.4 Telegramme neu.

**Reply by Hypnos3** [reply]
Der Vergleich zu Zigbee hinkt gewaltig. Zigbee läuft im selben Frequenzband wie WLAN (2,4 GHz). Für das 2,4GHz Band braucht man keine DutyCycle Anzeige, weil es dort schlicht keine DutyCycle Limitierung gibt.

Zigbee hat genauso wie WLAN mehrere Kanäle zur Auswahl (die selben wie bei WLAN nur anders nummeriert):
![image](https://user-images.githubusercontent.com/12692680/145393398-c8cd2db2-4267-4966-a3e3-ac24ed35019f.png)

Wenn ein Störsender einige Kanäle voll macht, kann man dort auf einen anderen Kanal ausweichen. Über die FritzBox hat man auch einen sehr guten Überblick über die vorhandenen Störsender, was einer CarrierSense Anzeige entspricht und modernere FritzBoxen versuchen zumindest belegten Kanälen automatisch auszuweichen.

Natürlich müssen Zigbee und WLAN koexistieren und das 2,4GHz Band ist schon sehr voll (weshalb man bei WLAN auf das 5GHZ Band ausgewichen ist). Am Ende hat man bei Zigbee die selben Probleme wie mit WLAN im 2,4GHz Band und da kann man sich überlegen ob man bei seinem Standort eher Probleme mit dem WLAN oder dem Homematic Funk hat.

**Comment by theimo1221** [comment, marked as answer]
Zusammenfassung:

1. Auch nach Downgrade/Update auf Beta/Nightly blieb das Problem bestehen
--> Fehler ist vermutlich Versionsunabhängig und ggf. einfach zeittechnisch Zufall
2. Hohe Carrier-Sense muss nicht am Hm System liegen, es kann auch schlicht die Sternenkonstellation oder Nachbarn sein.
--> Vielleicht behebt sich das Problem nach mehreren Tagen selbst (so scheinbar hier der Fall)
3. Vorgehen bei hohem Carrier-Sense. Alles akku-/stromlos schalten und falls danach alles passt, Gerät für Gerät wieder einschalten
4. Im Zweifelsfall mit passendem Analysetool den Störer suchen, oder Bundesnetzagentur einschalten
5. Diese Thematik ist ohne Wired-Verbindung nie komplett auszuschließen

Danke an @Baxxy13 und @jens-maus für die Unterstützung und Beistand 👍

**Reply by theimo1221** [reply]
![image](https://user-images.githubusercontent.com/20605452/145413477-9fe97d30-d374-440a-853a-13e8b8314e10.png)

Es geht nun seit 1 Stunde langsam kontinuierlich wieder hoch. Ich könnte mir mittlerweile vorstellen, dass es z.B. ein Auto der Nachbarn ist, was eben heute morgen weg war und nun wieder gekommen ist.

**Reply by jens-maus** [reply]
Hat ein Nachbar ein neues Elektroauto und hat es an die Ladesäule anzapft und die macht schönen Störfunk?

**Reply by jens-maus** [reply]
Ich würde vorschlagen du besorgst dir nen DVB-T Stick und dann ne Frequenz-Analyzer software und wanderst dann mal so durch die Umgebung um dem Störenfried auf den Grund zu gehen.

**Reply by theimo1221** [reply]
Konnte es Orten....
In einer Mülltonne hier war ein Heizungsaktor anno 2000 mit Batterie drin. Ohne Batterie war die Störung direkt beseitigt und der Spuk vorbei.

Deshalb auch Uhrzeitabhängig....

**Reply by jens-maus** [reply]
> Konnte es Orten.... In einer Mülltonne hier war ein Heizungsaktor anno 2000 mit Batterie drin. Ohne Batterie war die Störung direkt beseitigt und der Spuk vorbei.

Und was macht der Heizungsaktor in der Mülltonne? Sollte der entsorgt werden? Dann aber schön ordentlich über die Elektroschrott-Tonne entsorgen und ja, Batterien gehören ohnehin nicht in die Mülltonne.

**Reply by theimo1221** [reply]
> Und was macht der Heizungsaktor in der Mülltonne? Sollte der entsorgt werden? 

War in der Nachbarstonne, keine Ahnung, warum die den entsorgt haben. War aber auch nicht HomeMatic

**Reply by jens-maus** [reply]
Ah, dann war da in der Tat der Nachbar schuld. Interessant. Aber dann hätte sich das Problem sicher auch mit der nächsten Müllabfuhr erledigt gehabt.

**Reply by Baxxy13** [reply]
Sachen gibts... 
Was mich nun brennend interessiert... wie ortet man denn sowas in Nachbars Mülltonne?

**Reply by theimo1221** [reply]
> Was mich nun brennend interessiert... wie ortet man denn sowas in Nachbars Mülltonne? 

Mit Laptop und Antenne im Schutze der Dunkelheit durch die Nachbarschaft geistern. Nach rechts, ne hier ist schwächer, links wird's mehr, ne hier bin ich schon vorbei, ah hier wird's mehr. Hm hier vor der Hauswand bei den Mülleimern ist sehr hoch. Mal die Mülltonne wegziehen, ah jetzt ist es weg, zur Mülltonne gehen, hier ist wieder mehr, okay Handschuhe anziehen und wühlen😂

**Reply by jens-maus** [reply]
... und du hast noch vergessen: "Mit dem Übeltäter beim Nachbarn klingeln und sagen: 'Was ist das denn, bitte? So aber nicht'" ;-)

**Reply by blackbasket** [reply]
Hier das gleiche Problem :(

**Reply by MichaelN0815** [reply]
> Hier das gleiche Problem :(

Echt? Ist die Mülltonnen nun bei Dir angekommen?

**Comment by Lupus1969** [comment]
Moin,
nun muss ich auch einmal eine Frage stellen zum carrier Sense,
Ich betreibe ein RaspberryMatic auf einem Pi3
Nun passiert das 2x, dass mein Carrier Sense auf 100% geht, auf einmal, keine Änderung und kein Update. beim ersten mal war der Spuck nach etwa vier Wochen vorbei nun Knobel ich schon wider daran herum. Habe bis jetzt alle Geräte, sind etwa 150 Stück, stromlos geschaltet, trotzdem 100%. Nachdem ich den RaspberryMatic neu aufgesetzt habe war der Wert 0%, nach dem einspielen des Updates, sofort 100%. Alle Programme sind deaktiviert.

Ich hoffe, dass mir jemand die zündende Idee geben kann. Bin nicht so tief in der Materie, daher verzeiht etwaige Ungewissheit. wenn Ihr irgendwelche Infos benötigt, liefere ich sie gerne nach

**Reply by MichaelN0815** [reply]
CS bedeutet du hast einen Funkstörer. Siehe https://homematic-forum.de/forum/viewtopic.php?f=31&t=75532

Aber steht ja schon alles ausführlich hier.

**Reply by Lupus1969** [reply]
ja, diese ist richtig, habe den AskSin Analyzer XS am laufen und keine Fremdgeräte zu sehen oder erhöhter Sendestatus nur die Zentrale selber belegt etwa 50% der Tortenansicht...
